### DISCUSSION TITLE
Crab DNA

### DISCUSSION THREAD

**Discussion by tdgrossi** [coccoinomane/crabada.py#126] [Q&A] [answered]
Hi there!

I'm trying to determine a crab's parts by using the dna property, but it only displays a 75-length characters value. Does anyone knows how to `convert` it to actual body parts? e.g. pincers, mouth, dominant, recessive 1 and 2, etc? 

Hope it's not a moronic question, I'm not a python nor a web dev. Cheers!

**Comment by tdgrossi** [comment, marked as answer]
okay, stumbled upon my answer, https://docs.crabada.com/resources/community-resources/dna

**Comment by coccoinomane** [comment]
Thank you @tdgrossi for sharing your answer with us!
